src/Plugin/WebformElement/MineOrganisationsData.php

```diff
@@ -567,6 +567,10 @@ private function getSearchResult(array &$element, string $query): array {
    *   The user ids if any.
    */
   private function getSearchUserIds(string $query): array {
+
+    // Append wildcard character '*' to query string.
+    $query .= '*';
```

Comment:
I think you should check for existing wildcards before adding the default one.

Reply:
Added check for existing wildcard and removed potential whitespaces in 5eb32e678c2ed45a3b5d71d9434bef55d684f6ef.